samples/ingestion/ingestion-client/FetchTranscription/Language/AnalyzeConversationsProvider.cs

```diff
@@ -219,6 +220,43 @@ public async Task<bool> ConversationalRequestsCompleted(IEnumerable<AudioFileInf
             return errors;
         }
 
+        private static IEnumerable<ErrorEntity> GetAllErrorsFromResults((IEnumerable<AnalyzeConversationPiiResults> piiResults, IEnumerable<AnalyzeConversationSummarizationResults> summarizationResults, IEnumerable<string> errors)[] results)
+        {
+            var resultErrors = new List<ErrorEntity>();
+
+            if (results == null)
+            {
+                return resultErrors;
+            }
+
+            foreach (var result in results)
+            {
+                if (result.piiResults != null)
+                {
+                    foreach (var piiResult in result.piiResults)
+                    {
+                        if (piiResult.Errors != null)
```

Comment:
same here. add a null check here for the piiResult to be sure.

Reply:
updated to the code you shared below